cmd/get.go

```diff
@@ -28,6 +28,10 @@ var getCmd = &cobra.Command{
  For more documentation please see http://shopify.github.io/themekit/commands/#get
  `,
 	RunE: func(cmd *cobra.Command, args []string) error {
+		// This is a hack to get around theme ID validation for the list operation which doesnt need it
```

Comment:
What would the non-hacky fix be? Would it need changes on core?

Reply:
It would require making the validations of environments more variable, or requiring the user to input shop and password for this command only without it being able to be loaded from environment or config.